### PR TITLE
[bug-homepage] Fixed UI bugs

### PR DESCRIPTION
**Pull request by mojojain**
FIxed issues in homepage page. 

Test URLs:

- Before: https://main--mmsg-oly--aemsites.hlx.live/
- After: https://bug-homepage--mmsg-oly--aemsites.hlx.live/

### CODE REVIEW

File: blocks/savingscard/savingscard.css [hunk 1]
```diff
@@ -74,6 +78,18 @@
   margin-right: auto;
 }
 
+@media (width >= 768px) {
+  .block.savingscard > div.no-oly {
```
Comment (mrugesh-adobe) [inline blocks/savingscard/savingscard.css:82]: remove .block as selector just use .savingscard as selector. We know that @ManjunathJ-MMSG  worked on this. Do small fix along with this PR.

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -225,30 +225,66 @@
   font-size: var(--font-size-xs);
   line-height: 1.5rem;
 
+  .columns-wrapper .columns {
```
Comment (mrugesh-adobe) [inline blocks/columns/columns.css:228]: image
Observed horizontal scroll kindly fix it.

**Review by mrugesh-adobe — CHANGES_REQUESTED** [review]

File: blocks/savingscard/savingscard.css (hunk 1 above)
Comment (mojojain) [reply blocks/savingscard/savingscard.css]: Removed the selector @mrugesh-adobe

File: blocks/columns/columns.css (hunk 2 above)
Comment (mojojain) [reply blocks/columns/columns.css:228]: Fixed scrolling issue @mrugesh-adobe

**Review by mrugesh-adobe — APPROVED** [review]

**Pull request merged**